dlrm_data_pytorch.py

```diff
@@ -1061,8 +1061,8 @@ def write_dist_to_file(file_path, unique_accesses, list_sd, cumm_sd):
 if __name__ == "__main__":
     import sys
     import operator
-    import argparse
-
+    import argparse   
+    
```

Comment:
could we get rid of the extra whitespace?

Reply:
done!